Functions/TestResults.Tests.ps1

```diff
@@ -306,7 +306,8 @@ InModuleScope Pester {
             Export-NunitReport $testResults $testFile
             $xmlResult = [xml] (Get-Content $testFile)
             $xmlTestCase = $xmlResult.'test-results'.'test-suite'.'results'.'test-suite'.'results'.'test-case'
-            $xmlTestCase.name     | Should Be "Mocked Describe.Successful testcase"
+            # no Context for this test, so ".."
+            $xmlTestCase.name     | Should Be "Mocked Describe..Successful testcase"
```

Comment:
sure - changed the current tests to include a context (rather than checking for `..` it will check for the mocked context